webapp/channels/src/components/sidebar_right/sidebar_right.tsx

```diff
@@ -163,7 +163,16 @@ export default class SidebarRight extends React.PureComponent<Props, State> {
             // Focus the sidebar after a tick
             setTimeout(() => {
                 if (this.sidebarRight.current) {
-                    focusElement(this.sidebarRight, false);
+                    const rhsContainer = this.sidebarRight.current.querySelector('#rhsContainer') as HTMLElement;
+                    const searchContainer = this.sidebarRight.current.querySelector('#searchContainer') as HTMLElement;
+                    if (rhsContainer || searchContainer) {
+                        const firstFocusable = getFirstFocusableChild(rhsContainer || searchContainer);
+                        focusElement(firstFocusable || rhsContainer, true);
+                    } else {
+                        // Fallback: if rhsContainer isn't found, use sidebarRight.current directly.
```

Comment:
Why use the query selector first before falling back to using `sidebarRight.current`? Couldn't we just always use `sightbarRight.current`?

Reply:
Good idea. The RHS is definitely one of those areas that grew a ton, and we ended up repeating stuff slightly differently, so it makes sense to me to have a fallback case to cover all the variations that we might normally miss